.github/workflows/generate-package-list.yml

```diff
@@ -0,0 +1,72 @@
+name: generate-package-list
+on: 
+  workflow_dispatch:
+    inputs:
+      minorRelease:
+        description: 'Minor release (e.g. 1.13.x)'
+        required: true
+        type: string
+      patchRelease:
+        description: 'Patch release (e.g. 1.13.2)'
+        required: true
+        type: string
+      issueNumber:
+        description: 'Github issue number to close (e.g. 139)'
+        type: string
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  generate-package-list:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout docs site
+        uses: actions/checkout@v3
+        with:
+          path: bottlerocket-project-website
+      - name: Checkout bottlerocket os
+        uses: actions/checkout@v3
+        with:
+          repository: bottlerocket-os/bottlerocket
+          path: bottlerocket
+          ref: v${{ inputs.patchRelease }}
+      - name: Remove old package list
+        run: |
+          rm -rf bottlerocket-project-website/content/en/os/$MINOR_RELEASE/version-information/packages/$PATCH_RELEASE/
+          find bottlerocket-project-website/content/en/os/$MINOR_RELEASE/version-information/packages/
+        env:
+          MINOR_RELEASE: ${{ inputs.minorRelease }}
+          PATCH_RELEASE: ${{ inputs.patchRelease }}
+      - name: Make new patch release folder
+        run: |
+          mkdir bottlerocket-project-website/content/en/os/$MINOR_RELEASE/version-information/packages/$PATCH_RELEASE
+        env:
+          MINOR_RELEASE: ${{ inputs.minorRelease }}
+          PATCH_RELEASE: ${{ inputs.patchRelease }}
+      - name: Generate package version list
+        run: |
+          bash bottlerocket-project-website/scripts/software-versions-inventory/software-versions-inventory.sh bottlerocket/ > bottlerocket-project-website/content/en/os/$MINOR_RELEASE/version-information/packages/$PATCH_RELEASE/index.markdown
+        env:
+          MINOR_RELEASE: ${{ inputs.minorRelease }}
+          PATCH_RELEASE: ${{ inputs.patchRelease }}
+      - name: Open PR for changes
+        uses: peter-evans/create-pull-request@v5
+        with:
+          path: bottlerocket-project-website
+          commit-message: 'packages: add package versions for v${{ inputs.patchRelease }}'
+          branch: add-package-versions-${{ inputs.patchRelease }}
+          title: 'packages: add package versions for v${{ inputs.patchRelease }}'
+          body: |
+            **Issue number:**
```

Comment:
This seems to be a copy/paste of the PR template. I don't suppose there is a way to keep this synchronized with the actual PR template? (e.g. if I change the PR template, this workflow also changes)

Reply:
There doesn't seem to be a way to add parameters/input values when using a template file for the PR body.

Reply:
OK. It's a nice to have. Could you add a comment to `/.github/PULL_REQUEST_TEMPLATE` to warn anyone looking to change that file they'll also need to update this workflow?

Reply:
Yes - adding a comment to that template. Thanks!